Editor/PackageManager/DependencyManager.cs

```diff
@@ -11,7 +12,7 @@ namespace Innoactive.CreatorEditor.PackageManager
     /// Automatically retrieves all dependencies from the Unity's Package Manager at the startup.
     /// </summary>
     [InitializeOnLoad]
-    public static class DependencyManager
+    public class DependencyManager
```

Comment:
why?

Reply:
Because @SimonTheSourcerer is not happy about having a lot of static magic around.

Reply:
But all you did is kept everything static except the class definition, so you can create useless instances now 😕

Reply:
This class is never instantiated and is not intended to be instantiated. This and and `PackageOperationsManager` not only passed from static to normal classes but also the number of static methods were reduced and simplified in a more understandable way.